HeliosCLI/Makefile

```diff
@@ -34,6 +34,13 @@ ifneq ($(INCLUDES),)
     CFLAGS+=$(INCLUDES)
 endif
 
+# Define ROOT_OUTPUT_DIR and FINAL_TARGET
+ROOT_OUTPUT_DIR ?= ../output

```

Comment:
Don't output stuff to ../ put it in the current folder, makefiles generate things in the current folder not elsewhere